fastlane/Pluginfile

```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'trunk'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'libs-merge/source-id'
+# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.0'
```

Comment:
We need to release a new version of the release-toolkit then update this reference back to the new version once [this](https://github.com/wordpress-mobile/release-toolkit/pull/351) lands.